### PR TITLE
LPD-20187 Adapt Theme SVG Spritemap samples to support the new caching mechanism

### PR DESCRIPTION
**Pull request by izaera**

### CODE REVIEW

**Comment by izaera** [comment]
ci:test:relevant

File: workspaces/liferay-sample-workspace/client-extensions/liferay-sample-theme-spritemap-1/client-extension.yaml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 assemble:
     - from: assets
+      hashify: spritemap.svg
```
Comment (dsanz) [inline workspaces/liferay-sample-workspace/client-extensions/liferay-sample-theme-spritemap-1/client-extension.yaml:3]: is this some sort of black magig? wasn't aware of it. For single files not imported elsewhere, this is a nice option!

**Comment by dsanz** [comment]
LGTM ;)

File: workspaces/liferay-sample-workspace/client-extensions/liferay-sample-theme-spritemap-1/client-extension.yaml (hunk 1 above)
Comment (izaera) [reply workspaces/liferay-sample-workspace/client-extensions/liferay-sample-theme-spritemap-1/client-extension.yaml:3]: It was introduced in the favicon PR when I suggested to create a task to hashify files. Brian made it with Peter Shin.

**Comment by izaera** [comment]
ci:foward:force

**Comment by izaera** [comment]
ci:foward:force

**Comment by izaera** [comment]
Since CI is ingnoring me I'll forward it manually :shrug:

**Pull request closed without merging**